### ISSUE TITLE
Imported variables that used in string switch are not imported correctly in output

### ISSUE THREAD

**Issue by seanchas116** [Microsoft/TypeScript#5645]
I experienced the following problem:

## Problem

Variables that are imported with `import` statement and used in `switch` statement with string parameter 
are not imported correctly in emitted JavaScript (no import statement emitted for those variables).

Using other types than string in `switch` does not cause the problem oddly.

## Version

Version 1.8.0-dev.20151112

### mod.ts

```ts
export const foo = 1;
export const bar = "2";
```

### test.ts

```ts
import {foo, bar} from "./mod";

switch ("foo") {
  case "foo": {
    foo;
  }
  case "bar": {
    bar;
  }
}
```

## Command

```
tsc test.ts --target es6
```

## Output

tsc does not generate `import` statement for `foo` and `bar`.

### test.js

```js
switch ("foo") {
    case "foo":
        foo;
    case "bar":
        bar;
}
```

**Comment by seanchas116** [comment]
This problem seems to be occurred since `1.8.0-dev.20151110`.

**Comment by seanchas116** [comment]
I noticed whole type checking doesn't work inside `switch` (any wrong code inside `switch` compiles without errors)...

**Comment by seanchas116** [comment]
Oh this is fixed in #5639
